### PR TITLE
Fx2trt converters

### PR DESCRIPTION
**Pull request by apbose**
# Description

This PR addresses the feature[ #1657](https://github.com/pytorch/TensorRT/issues/1657)

## Type of change
Addition of the following files-

1. fx/converters/fx2trt_converters.py
Converters added- 
- torch.ops.aten.add.Tensor
- torch.ops.aten.leaky_relu
- torch.ops.aten.adaptive_avg_pool2d.default
- torch.ops.aten._adaptive_avg_pool3d.default
- torch.ops.aten.mean.dim
- torch.ops.aten.batch_norm
- torch.ops.aten.cat.default
- torch.ops.aten.convolution.default

2. fx/converters/fx2trt_converters_util.py

3. Tests-
- torch.ops.aten.add.Tensor
Added a seperate test_add_torch2trt.py for testing. Will remove this later, since this is already covered in test_binary_ops_aten.py
- torch.ops.aten.leaky_relu
Added test_leaky_relu_torch2trt.py
- torch.ops.aten.adaptive_avg_pool2d.default
Already present
- torch.ops.aten._adaptive_avg_pool3d.default
Already present
- torch.ops.aten.mean.dim
Already present
- torch.ops.aten.batch_norm
Already present
- torch.ops.aten.cat.default
Already present
- torch.ops.aten.convolution.default
Already present

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

**Comment by apbose** [comment]
Raised #1745
